llvm/docs/DeveloperPolicy.rst

```diff
@@ -46,7 +46,7 @@ quality.
 Stay Informed
 -------------
 
-Developers should stay informed by reading the `LLVM Discourse forums`_. 
+Developers should stay informed by reading the `LLVM Discourse forums`_.
 If you are doing anything more than just casual work on LLVM, it is suggested that you also
 subscribe to the "commits" mailing list for the subproject you're interested in,
 such as `llvm-commits
```

Comment:
We should mention the new teams to subscribe to and link to the CODEOWNERS file I think.